fixtures/higherhealth_campuses.csv

```diff
@@ -349,3 +349,4 @@ ZA-WC,West Coast,Citrusdal
 ZA-WC,West Coast,Malmesbury
 ZA-WC,West Coast,Vredenburg
 ZA-WC,West Coast,Vredendal
+,Other,Other

```

Comment:
I still don't like the idea of putting this in the database. I would really prefer this to all be handled on the front-end.

Reply:
If we are going to have to put a strange value like this in the database, then we should create a data migration for it, to ensure that it exists.

We should also add something to the model to prevent us from ever deleting it.

Reply:
I don't think that this has been addressed yet